.github/workflows/ci.yml

```diff
@@ -122,3 +122,24 @@ jobs:
         run: |
           export PATH=$HOME/.cargo/bin:$HOME/.local/bin:$HOME/go/bin:$PATH
           make ci_gha
+
+      - name: Upload bitcoind log
+        if: failure()
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: e2e-logs-bitcoind.log
+          path: api_tests/log/bitcoind/regtest/debug.log
+
+      - name: Upload parity log
+        if: failure()
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: e2e-logs-parity.log
+          path: api_tests/log/parity/parity.log
+
+      - name: Upload e2e logs
+        if: failure()
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: e2e-test-logs
+          path: api_tests/log/tests/
```

Comment:
Did you consider just uploading the `api_test/log` folder? And if so, why deciding not to upload it entirely?

Reply:
I did consider it and decided against it due to it's size: `api/log` contains `api/log/parity` and `api/log/bitcoind` which can get rather big. These folder contain the whole blockchain data. 

--> update the PR description.